TestCases/compliance-level-3/0087-chapter-11-example/0087-chapter-11-example.dmn

```diff
@@ -248,7 +248,7 @@
 	</semantic:decision>
 	<semantic:businessKnowledgeModel id="_dc2ca64d-2044-4806-9d0e-e705b3b14447" name="Eligibility rules">
 		<semantic:description>&lt;p&gt;&lt;span style="font-family: arial, helvetica, sans-serif; font-size: 10pt;"&gt;&lt;span lang="JA"&gt;The&amp;nbsp;&lt;/span&gt;&lt;strong&gt;&lt;span lang="JA"&gt;Eligibility rules&amp;nbsp;&lt;/span&gt;&lt;/strong&gt;&lt;span lang="JA"&gt;decision logic&amp;nbsp;defines a complete, priority-ordered single hit decision table&amp;nbsp;&lt;/span&gt;deriving Eligibility from Pre-Bureau Risk Category, Pre-Bureau Affordability and Age.&lt;/span&gt;&lt;/p&gt;</semantic:description>
-		<semantic:variable name="Eligibility rules" id="_fe2d2049-704b-4a88-88be-3d1a56c3d376" typeRef="tEligibility"/>
```

Comment:
@opatrascoiu - this is the example from the spec.  My feeling is removing these types from the variables throughout changes the nature of the bkms from one where a variable value may be type-coerced due to the typing on the variable, into one where the variable is type-inferred.  So, whereas with the typings in place, there is a possibility that a null value will be returned because of null-coercion forced by the explicit type.  With the types removed this is no longer the case.

Reply:
I know the example is from the spec, but in my opinion is incorrect. I raised an issue to be discussed in RTF.

Currently is in INBOX
https://issues.omg.org/browse/INBOX-1081

Reply:
Hi @opatrascoiu .  That link requires a login I am afraid.  Though, for the benefit of those without login creds there, I think this is it: https://issues.omg.org/issues/lists/unclassified#issue-47325.  I'm not sure the position of the typeRef is incorrect according to that ticket, having a typeRef on the variable is permissible, but if the typedef of an invokable is supposed to be function type then yes, I get your point.   Maybe leave the typedef where it is, but make them function types? (as per one of the options)